packages/documentation-site/patternfly-docs/content/accessibility/tree-view/tree-view.md

```diff
@@ -3,32 +3,89 @@ id: Tree view
+| `aria-labelledby="[id of the element that labels the tree view]"` | `ul.pf-v5-c-tree-view__list` | Adds an accessible name to the tree view. **Required** if the `aria-label` attribute is not passed in. |
+| `aria-multiselectable="true"` | `ul.pf-v5-c-tree-view__list[role="tree"]` | Indicates that multiple tree view items can be selected. **Required** if the tree view is intended to allow the selection of multiple items.<br/>Note that passing this prop simply indicates that the tree view is multiselectable, and any selection logic must still be handled manually. |
+| `role="group"` | `ul.pf-v5-c-tree-view__list` | Identifies the element as a group of nested tree view items. **Required** `ul` elements nested within the root `ul[role="tree"]`. |
+| `role="tree"` | `ul.pf-v5-c-tree-view__list` | Identifies the element as a tree widget. **Required** on the root level `ul` of the tree view only. |
+| `aria-expanded="false"` | `li.pf-v5-c-tree-view__list-item` | Indicates that the tree view item is collapsed to assistive technologies, and that its children are not visible nor navigable. **Required** if a tree view item is collapsed and it has children nodes. |
+| `aria-expanded="true"` | `li.pf-v5-c-tree-view__list-item.pf-m-expanded` | Indicates that the tree view item is expanded to assistive technologies, and that its children are visible and navigable. **Required** if a tree view item is expanded and it has children nodes. |
+| `aria-selected="false"` | `li.pf-v5-c-tree-view__list-item` | Indicates that a tree view item is not selected. **Required** if a tree view item is not selected. |
+| `aria-selected="true"` | `li.pf-v5-c-tree-view__list-item` | Indicates that a tree view item is currently selected. **Required** if a tree view item is selected. |
+| `role="treeitem"` | `li.pf-v5-c-tree-view__list-item` | Identifies the element as a group of nested tree view items. **Required** `ul` elements nested within the root `ul[role="tree"]`. |
+| `tabindex="-1"` | `button.pf-v5-c-tree-view__node, button.pf-v5-c-tree-view__node-toggle, button.pf-v5-c-tree-view__node-text, .pf-v5-c-tree-view__node-check > input[type="checkbox"], .pf-v5-c-tree-view__action > button` | Makes the element focusable without including it in the tab sequence of the page. **Required** on all tree items, toggles, checkboxes, and action buttons, except for the element that is currently or was last focused within the tree view. This must be updated as a user navigates through a tree view. |
+| `tabindex="0"` | `button.pf-v5-c-tree-view__node, button.pf-v5-c-tree-view__node-toggle, button.pf-v5-c-tree-view__node-text, .pf-v5-c-tree-view__node-check > input[type="checkbox"], .pf-v5-c-tree-view__action > button` | Includes the element in the tab sequence. **Required** on the tree item, toggle, checkbox, or action button that is currently or was last focused within the tree view. This must be updated as a user navigates through a tree view. |
```

Comment:
The core docs currently (but once my Core PR goes in, previously) mentioned the tabindex for the `li[role="treeitem"]` element. This matches what WAI examples are doing, like their [file tree view example](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-1a/).

However, what the React implementation is doing is updating the tabindex on a button element that is inside these li elements:

![image](https://github.com/patternfly/patternfly-org/assets/70952936/4639b0c9-d652-4731-9ebb-d6c6b7d2e6e1)

Or at least that's the intent when the examples get more complicated (separate expansion/selection, checkboxes, and actions).

I included all of the elements that could receive focus in a tree view above, but since the examples in React don't currently work that way (I opened an issue regarding this, it may be a bit complicated to decide how we want to go with it), I could just mention the tree view nodes here for now, e.g. the `button.pf-v5-c-tree-view__node` elements in the most basic examples.

That said, do we want to also update core/react implementations to omit the tabindex on the li elements?